src/main/scala/com/mozilla/telemetry/stats/MannWhitneyU.scala

```diff
@@ -49,8 +51,8 @@ package object stats {
     }
   }
 
-  def tieCorrect(sample: Map[Int, Int]): Double = {
-    val n = sample.foldLeft[Int](0)(_ + _._2)
+  def tieCorrect(sample: Map[Long, Long]): Double = {
+    val n = sample.foldLeft[Long](0)(_ + _._2)
```

Comment:
Might want to change the starting value to 0L for clarity here and and other places you foldLeft

Reply:
Good find, thank you.